app/models/tag.rb

```diff
@@ -0,0 +1,4 @@
+class Tag < ActiveRecord::Base
+ validates :name, presence: true
```

Comment:
probably also want to validates for uniqueness too here.
since we don't want really need any duplicate tags